pkg/console/operator/sync_v400.go

```diff
@@ -123,10 +124,23 @@ func sync_v400(co *consoleOperator, originalOperatorConfig *operatorv1.Console,
 		}
 	}
 
-	// the operand is available if all resources are present & if we have all the replicas
+	// the operand is available if all resources are:
+	// - present
+	// - if we have all the replicas
+	// - route is admitted
 	// available is currently defined as "met the users intent"
-	if deploymentsub.IsReady(actualDeployment) && routesub.IsAdmitted(rt) {
-		co.ConditionDeploymentAvailable(operatorConfig)
+	if deploymentsub.IsReady(actualDeployment) {
```

Comment:
nit: I have a slight preference to avoid the nesting

```go
if !deploymentsub.IsReady(actualDeployment) {
  co.ConditionDeploymentNotAvailable(operatorConfig)
} else if !routesub.IsAdmitted(rt) {
  co.SetStatusCondition(...)
} else {
  co.ConditionDeploymentAvailable(operatorConfig)
}
```